packages/rancher-istio/charts/templates/istio-cni-psp.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: psp-istio-cni
+  namespace: {{ template "istio.namespace" . }}
+spec:
+  allowPrivilegeEscalation: true
+  fsGroup:
+      rule: RunAsAny
+  hostNetwork: true
+  runAsUser:
+      rule: RunAsAny
+  seLinux:
+      rule: RunAsAny
+  supplementalGroups:
+      rule: RunAsAny
+  volumes:
+  - secret
+  - configMap
+  - emptyDir
+  - hostPath
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: psp-istio-cni
```

Comment:
Wondering if we should expose this in `values.yaml`
(and its references throughout the other file(s))

Reply:
w.r.t. naming, I think we just need to introduce a helper template instead of hard-coding the name, re: my comment in https://github.com/rancher/charts/pull/957#pullrequestreview-615773317. Not worth blocking this PR for though.

What else would you want to expose in the values.yaml?

Reply:
I'm hesitant to open up values.yaml fields unless absolutely necessary since it just adds more stuff we need to test / support per release.

Reply:
Agreed on not blocking the PR for it, but the `template` may suffice. Agreed on the `values.yaml` though; just want to anticipate potential tech debt if naming is a bottleneck or a user wants to bring their own RBAC (unlikely)